.rubocop_local.yml

```diff
@@ -3,3 +3,14 @@ GlobalVars:
   # Loggers
   - $lenovo_log
   - $log
+
+# Ignores the file at lib folder that needs to have a naming with dashes.
+AllCops:
+  Exclude:
+  - lib/manageiq-providers-lenovo.rb
+
+# Forcing Single-quote string literals except in interpolations
+Style/StringLiterals:
+  Enabled: true
```

Comment:
For consistency we choose at the ManageIQ common level to not allow forcing this.  In fact, many of us would have preferred double-quote only, but as a community we've chosen to not to enforce either side.

Reply:
Ok, Jason. Good point.